website_cloudfront.tf

```diff
@@ -1,25 +1,11 @@
-data "terraform_remote_state" "lambda-api" {
-  backend   = "s3"
-  workspace = local.workspace
-
-  config = {
-    endpoint             = "s3.amazonaws.com"
-    key                  = "lambda-api.tfstate"
-    workspace_key_prefix = "nata/dia3"
-    bucket               = "jr-terraform-states"
-    region               = "us-east-1"
-    encrypt              = true
-  }
-}
-
 # TNA provided this certificate upon our request, importing it into the three
 # development environments on our behalf.
 # Note that SSL certificates for use with CloudFront _must_ be imported into
 # the US East (N. Virginia) Region (us-east-1).
 data "aws_acm_certificate" "wildcard" {
-  domain   = "*.nationalarchives.gov.uk"
+  domain   = local.service[terraform.workspace].url
   statuses = ["ISSUED"]
-  types    = ["IMPORTED"]
+  types    = ["AMAZON_ISSUED"]
   provider = aws.us-east-1
```

Comment:
Is this region correct?

Reply:
Yeah, certificates for Cloudfront need to go in us-east-1